documentation/release-notes/source/2019.2.rst

```diff
@@ -24,6 +56,20 @@ Run-time
   significanly improves the performance of code using the LLVM
   back-end.
 
+* Support for ``apply-mep`` method entry was added to the LLVM
+  run-time.
```

Comment:
Add a sentence about the benefits of this?  I assume it's a speed-up, not a bug fix.